applications/nrf_desktop/configuration/common/led_effect.h

```diff
@@ -68,8 +113,19 @@ struct led_effect {
 		.loop_forever = false,						\
 	}
 
+
+/** Create LED turned off effect initializer.
+ */
 #define LED_EFFECT_LED_OFF() LED_EFFECT_LED_ON(LED_NOCOLOR())
 
+
+/** Create LED blinking effect initializer.
+ *
+ * LED color is periodically changed between selected color and LED turned off.
```

Comment:
```suggestion
 * LED color is periodically changed between the selected color and the LED turned off.
```

Reply:
Not really a fan of `LED turned off` and `LED turned on` as names of states. Are there any other options?

Reply:
We already had `LED_EFFECT_LED_OFF` an `LED_EFFECT_LED_ON` as macro names. I wanted to stick with macro names. Otherwise it could be ambigous. And I have no idea of better name for the macros.